src/coreclr/md/compiler/custattr_emit.cpp

```diff
@@ -991,9 +991,7 @@ HRESULT RegMeta::_IsKnownCustomAttribute(        // S_OK, S_FALSE, or error.
 
     // Add to hash.
     sLookup.ca = ixCa;
-    pFound = m_caHash.Add(&sLookup);
-    IfNullGo(pFound);
-    *pFound = sLookup;
+    m_caHash.Add(sLookup);
```

Comment:
Do we need EX_TRY/EX_CATCH around this?

Reply:
Ah, I'll make it `AddNoThrow` and return if it doesn't succeed. That should match the old behaviour of `IfNullGo(pFound)`.